### ISSUE TITLE
Swap `SECRET` and `API_KEY` ?

### ISSUE THREAD

**Issue by wi-wissen** [FriendlyCaptcha/friendly-lite-server#9]
I don't know if it's just me, but from my point of view `SECRET` and `API_KEY` are used against the other convention.

Currently, you define `SECRET`, which you use as the `sitekey` in the widget. And when checking, the result the `API_KEY` is the parameter `secret` but then `SECRET` is the parameter `sitekey`.

Wouldn't it make sense to swap the variables in the lite server?

**Comment by amenk** [comment]
I think you are right. We could also just call it sitekey and so on

**Comment by lhaemmerle** [comment]
Yes, fwiw I found the naming of the parameters/constants confusing as well :-)
